### ISSUE TITLE
Always show sidebar on PC

### ISSUE THREAD

**Issue by adeebshihadeh** [commaai/new-connect#73]

**Comment by griffinbaker12** [comment]
even when the screen gets reduced? i feel like the old connect site had it right: default open, but if the screen collapses then move it into the sidebar and have it hover over content when expanded. lmk though and I can fix.

**Comment by adeebshihadeh** [comment]
Yeah, just like old connect. Open on big screens, and same behavior as now on small ones.

**Comment by 35C4n0r** [comment]
@adeebshihadeh I raised a PR for this, can you confirm if this is what we wanted?

**Comment by ugtthis** [comment]
Came up with a design idea that is supposed to be on the leaner side with minimum changes to the overall new-connect UI

### Thoughts:
- A header component(the black bar that runs across at the top) gives the layout a clearer hierarchy. Without the top header component, it looks like there are 2 headers. 
- Having the device name in multiple places close to each other when on desktop mode is redundant because sidebar will be open already displaying the current device that is being displayed
- When on desktop users should not have an option to collapse the sidebar.  Only when on mobile will users be able to open and close the sidebar

<br>

### new-connect with the sidebar as is open
<img width="730" alt="no-header-component" src="https://github.com/user-attachments/assets/9fe2a345-9aeb-4d98-95f3-9ceca62421aa">

<br>

### new-connect with a header component
<img width="529" alt="header-component-1" src="https://github.com/user-attachments/assets/c4f4caec-a3df-4446-9b9a-d16c5c4a33a9">


<br><br>

# Design
![sidebar-desktop](https://github.com/user-attachments/assets/353a2286-2d4e-4eaf-b5ee-cee053059a0c)
